contributor-docs/migrating-to-css-modules.md

```diff
@@ -0,0 +1,70 @@
+# Primer React Styled Components to CSS Modules Refactoring Guide
+
+This guide outlines the steps to follow when refactoring Primer React components from Styled Components to CSS Modules.
+
+---
+
+## Table of Contents
+
+- [Primer React Styled Components to CSS Modules Refactoring Guide](#primer-react-styled-components-to-css-modules-refactoring-guide)
+  - [Table of Contents](#table-of-contents)
+  - [Before Refactoring a Component](#before-refactoring-a-component)
+  - [How to Migrate](#how-to-migrate)
+    - [Refactoring Styled-Components to CSS Modules](#refactoring-styled-components-to-css-modules)
+    - [When Refactoring a Component](#when-refactoring-a-component)
+    - [Testing the Migration](#testing-the-migration)
+  - [Releasing a Component](#releasing-a-component)
+
+---
+
+## Before Refactoring a Component
+
+- **Verify VRT (Visual Regression Testing) Coverage:**
+  - Check for missing VRT coverage. We utilize the VRT tests to make sure we're matching styling in production with current expectations.
+  - Make sure there are `dev` stories for any edge cases spotted in production for the component (ie. sx prop, custom className, styled system attributes).
+- **Ensure All Visual Changes Are Completed:**
+  - Make necessary visual changes **before** creating the CSS Modules refactor PR.
+
+---
+
+## How to Migrate
+
+### Refactoring Styled-Components to CSS Modules
+
+- **Replace `${get('...')}` Syntax:**
+  - Migrate these to CSS Modules classes.
```

Comment:
Do you mean CSS variables here? I would link to https://primer.style/foundations/primitives/color and then include a code example showing the conversion